### PR TITLE
 linuxkit: Update linuxkit to v0.8+

### PR DESCRIPTION
**Pull request by lexnv**
The linuxkit blob is updated from the upstream branch
with commit 46d4edc [1].

The blob is slight modified to support prefixing the
filesystem of the docker image. This is necessary for
generating the initrd image.

To avoid size limitations present in the linux kernel,
the gzip wrapper is removed.

### X86_64
- `file ./blobs/x86_64/linuxkit`

```
[Before] dynamically linked, stripped
[After]  statically linked, stripped
```

- file size
```
[Before] 57M
[After]  48M
```

### Aarch64

- `file ./blobs/aarch64/linuxkit`

```
[Before] dynamically linked, stripped
[After]  statically linked, stripped
```

- `file size: 55M`

```
[Before] 55M
[After]  45M
```

[1] https://github.com/linuxkit/linuxkit/commit/46d4edc967b5a9de705840d30c83b83fedabb492

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by petreeftime** [comment]
Instead of pushing this to the repo, can we put it somewhere in S3 instead?

File: tests/tests.rs [hunk 1]
```diff
@@ -87,15 +87,15 @@ mod tests {
         .1;
         assert_eq!(
             measurements.get("PCR0").unwrap(),
-            "93a8a6a775cd1e1ab8b6121d1b0ce08e99e0976dabfa40663fa8ea9633421305de18c8f95aa2b82d3feb918fe912c838"
+            "a5a11ad2b3a60117c14b9fe1278b4b902ac0407e76a7b2d7a1f9e60dbacbd8a621acfe080c834c291fd13b348af0c5b1"
```
Comment (petreeftime) [inline tests/tests.rs:90]: Can you explain why the PCRs changed here? I don't think rebuilding the same container image should ever change the PCRs.

**Review by petreeftime — CHANGES_REQUESTED** [review]
The build tool should not influence the PCR values. If the internal layout changed, we should fix it, such that the update is transparent to users.

File: tests/tests.rs (hunk 1 above)
Comment (lexnv) [reply tests/tests.rs:90]: I'll have a look at the changes from upstream to determine a viable path forward for updating the linuxkit. Indeed, the PCRs should not change

Comment (shtaked) [reply tests/tests.rs]: There are 2 changes I see in the `.img` file produced by newer version of linuxkit:
* the output is `gzip`ped with the same set of arguments we used before
* the content of the image changed - there are some duplicate entries for `rootfs/etc/hosts` and `rootfs/etc/resolv.conf`

**Comment by shtaked** [comment]
Updating the Linuxkit version won't fix the "Out of memory" issues and doesn't worth the effort. Closing the PR.

**Pull request closed without merging**